bip-0085.mediawiki

```diff
@@ -207,7 +221,9 @@ OUTPUT:
 ===HD-Seed WIF===
 Application number: 2'
 
-Uses 256 bits[1] of entropy as the secret exponent to derive a private key and encode as a compressed WIF which will be used as the hdseed for Bitcoin Core wallets.
+Uses the most significant 32 bytes[1] of entropy as the secret exponent to derive
```

Comment:
You can define references inline as follows

```suggestion<ref name="multiple" />
Uses the most significant 32 bytes<ref name="curve-order">There is a very small chance that you'll make an invalid key that is zero or bigger than the order of the curve. If this occurs, software should hard fail (forcing users to iterate to the next index).</ref> of entropy as the secret exponent to derive
```

and then replace the Footnotes section at the bottom with 

```
==Footnotes==

<references />
```